include/multipass/id_mappings.h

```diff
@@ -33,24 +35,46 @@ namespace multipass
 {
 using id_mappings = std::vector<std::pair<int, int>>;
 
-inline id_mappings unique_id_mappings(const id_mappings& xid_mappings)
+inline auto unique_id_mappings(id_mappings& xid_mappings)
```

Comment:
I am not a fan of this in a header, but this whole file shouldn't even exist (this function is an implementation detail of `VMMount` exclusively). Fixing that is out of scope here, so fine.